packages/module-backup/src/client/AutoBackup.tsx

```
@@ -0,0 +1,515 @@
+    <>
+      <ButtonComponent icon={<PlusOutlined />} type="primary" onClick={showModal}>
+        {t('New Auto')}
+      </ButtonComponent>
+      <Modal
+        title={t('New Auto')}
+        width={800}
+        open={isModalOpen}
+        // onOk={handleOk}
+        onCancel={handleCancel}
+        footer={[
+          <Row gutter={16} justify="end" align="middle">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)